magento/adobe-stock-integration
# Redirect URI pattern with escaped hostname

### Description (*)
Adobe I/O console requires to have the periods (.) in the hostname escaped for the "Redirect URl pattern".
 
![image](https://user-images.githubusercontent.com/2148959/63175682-28bcfe00-c04d-11e9-96d4-4088b94378aa.png)

Currently, for the "Redirect URL pattern" the system returns the same value as for the "Redirect URL". You need to modify the value manually by escaping the periods in the hostname when you paste the pattern value on the Adobe I/O console side. 

This PR brings a change that escapes the periods from hostname of the "Redirect URL pattern" automatically, so the user can easily copy the value from admin panel and paste to Adobe I/O without additional manual edit. 

Please note that only the hostname part must be escaped. 

### Fixed Issues (if relevant)
N/A

### Manual testing scenarios (*)
1. Go to Admin -> Stores -> Configuration -> System -> Adobe Stock Integration
2. Make sure that values for Redirect URI and Pattern are not the same and the periods in the hostname in the Pattern are escaped.

![image](https://user-images.githubusercontent.com/2148959/63176059-e47e2d80-c04d-11e9-93f1-1fee52107b87.png)
